### ISSUE TITLE
[BUG] UA_ClientConfig_setDefaultEncryption() does not accept an issuerList

### ISSUE THREAD

**Issue by costinzanfir** [open62541/open62541#6912]
## Description
The UA_ClientConfig_setDefaultEncryption() does not accept an issuerList as argument.

## Background Information / Reproduction Steps
When a certificate signed by a CA is being used by the server, this certificate must be added to the issuerList of the client. In version 1.4.6, UA_ClientConfig_setDefaultEncryption() does not permit this (UA_CertificateVerification_Trustlist() is being called with issuerList set to NULL).

Self-signed certificates are working fine.

There is a workaround: call UA_CertificateVerification_Trustlist() explicitly in the client code.
```c
UA_StatusCode retval = UA_ClientConfig_setDefaultEncryption(cc,
    certificate, private_key,
    issuer_list, issuer_list_size,
    NULL, 0);
...
retval = UA_CertificateVerification_Trustlist(
    &cc->certificateVerification,
    issuer_list, issuer_list_size,
    issuer_list, issuer_list_size,
    NULL, 0);
```

I know that UA_ClientConfig_setDefaultEncryption() is meant as an example, but it would be nice to have this option.

Thanks!

Used CMake options:

```bash
cmake -DBUILD_SHARED_LIBS=yes -DUA_ENABLE_NODEMANAGEMENT=yes -DUA_ENABLE_METHODCALLS=yes -DUA_ENABLE_SUBSCRIPTIONS=yes -DUA_ENABLE_PUBSUB=no -DUA_ENABLE_PUBSUB_INFORMATIONMODEL=no -DUA_NAMESPACE_ZERO=FULL -DUA_ENABLE_MQTT=OFF -DUA_ENABLE_ENCRYPTION=OPENSSL -DUA_ENABLE_ENCRYPTION_OPENSSL=ON
```

## Checklist

Please provide the following information:

 - [X] open62541 Version (release number or git tag): 1.4.6
 - [X] Other OPC UA SDKs used (client or server): none
 - [X] Operating system: Ubuntu 22
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by costinzanfir** [comment]
Hi,

The issue is solved in 1.4.8.

Thanks!

**Comment by costinzanfir** [comment]
Hi,

The issue is solved in 1.4.8.

Thanks!
